easybuild/tools/systemtools.py

```diff
@@ -998,6 +1003,106 @@ def get_glibc_version():
     return glibc_ver
 
 
+def get_cuda_object_dump_raw(path):
+    """
+    Get raw ouput from command which extracts information from CUDA binary files in a human-readable format,
+    or None for files containing no CUDA device code.
+    See https://docs.nvidia.com/cuda/cuda-binary-utilities/index.html#cuobjdump
+    """
+
+    res = run_shell_cmd("file %s" % path, fail_on_error=False, hidden=True, output_file=False, stream_output=False)
+    if res.exit_code != EasyBuildExit.SUCCESS:
+        fail_msg = "Failed to run 'file %s': %s" % (path, res.output)
+        _log.warning(fail_msg)
+
+    # check that the file is an executable or object (shared library) or archive (static library)
+    result = None
+    if any(x in res.output for x in ['executable', 'object', 'archive']):
+        # Make sure we have a cuobjdump command
+        if not shutil.which('cuobjdump'):
```

Comment:
This is already checked at https://github.com/easybuilders/easybuild-framework/pull/4692/files#diff-00260ae7a519d5825760f53b067b29fb84a3e0d2649e6a27ace99abaca96d7d1R4361 is this required in both places?